packages/near-membrane-dom/.rolluprc.cjs

```diff
@@ -1,7 +1,46 @@
 'use strict';
 
-const { rollupConfig } = require('@locker/scripts/rollup/configs/base.cjs');
+const fs = require('fs-extra');
+const path = require('path');
+const replacePlugin = require('@rollup/plugin-replace');
+const { createConfig, rollupConfig } = require('@locker/scripts/rollup/configs/base.cjs');
 
-module.exports = rollupConfig({
-    external: ['@locker/near-membrane-base'],
-});
+const customDevtoolsFormatterBasename = 'custom-devtools-formatter';
```

Comment:
Oh, when you said it would be exposed as a module, I thought it would be a module published in `near-membrane-base/custom-devtools-formatter` and I can use it by `import { enableChromeDevtools } from "near-membrane-base/custom-devtools-formatter"`.

The current way I need to install `near-membrane-dom` to use it.

Reply:
Correct. You'll need to install `near-membrane-dom`. However, it's packaged so that you can load it without requiring the rest of the package (so no iframe).